### DISCUSSION TITLE
ImageMagick can't read font courier?

### DISCUSSION THREAD

**Discussion by IfgiU** [ImageMagick/ImageMagick#6169] [Help] [answered]
### ImageMagick version

7.1.1-3

### Operating system, version and so on

Zorin OS 16 (Ubuntu 20.04)

### Description

Hi!
I'm using the python library [MoviePy](https://zulko.github.io/moviepy/), that uses ImageMagick as an optional dependency for some functions. So, I downloaded the portable version of ImageMagick for linux, configured the path to it in MoivePy, and tried to use some functions. However, I get this error:
```
magick: unable to read font `Courier' @ warning/annotate.c/RenderType/1014.
magick: unable to read font `Courier' @ error/annotate.c/RenderFreetype/1622
```
Now, MoviePy puts directly under these errors a note saying that I maybe messed up configuring the path to the executable. I double checked and am pretty sure that the path is correct. Also, the error comes from ImageMagick, doesn't it? So that's why I decided to ask here and not on MoviePy.

**Comment by fmw42** [comment]
What do you get from 

`magick -version`

Does it include freetype?

Do you have Courier font on your system?

Did the command you used reference the font courier by name or by its path to the font file?

**Comment by snibgo** [comment]
Try `magick -list font` to see what fonts IM can find. For example, on my Windows laptop, that font is named `Courier-New`.`

**Comment by IfgiU** [comment, marked as answer]
Hi, thanks for helping! Turns out, I am stupid. The function I was trying to use in MoviePy had the Courier font set as a default, and I didn't overwrite that, so it just used courier, which wasn't installed on my computer.
